src/main/java/svc/managers/ViolationManager.java

```diff
@@ -13,12 +13,59 @@
 public class ViolationManager {
 	@Inject
 	private ViolationDAO violationDAO;
+	@Inject
+	private SMSManager smsManager;
```

Comment:
Again - Same as CitationManager - SMS Manager should utilize ViolationManager, ViolationManager shouldn't know about SMSManager. Think about SMSManager being at a higher level than these domain managers. 

SMSController
SMSManager
Domain Managers (Citation, Violation, etc.)
DAOs/REST calls to other services